cartridge/roles.lua

```diff
@@ -467,10 +484,14 @@ local function stop()
         if (service_registry.get(role.role_name) ~= nil)
         and (type(role.M.stop) == 'function')
         then
+            log.info('Stop "%s" role', role.role_name)
+            local start_time = fiber.time64()
```

Comment:
Let's use clock.monotonic64 instead of fiber.time64, because fiber.time64 is the same until yield or log happens.